examples/print_tree.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+'''
+This script demonstrates using the AtramhasisProvider to print a full thesaurus
+as a simple rst tree. The generated tree can be saved as a file and turned into
+a pdf or imported in a python Sphinx documentation project.
+'''
+
+from skosprovider_atramhasis.providers import AtramhasisProvider
+
+import requests
+
+# Use to only generate the tree for certain top terms.
+# TOP_CONCEPTS = None
+TOP_CONCEPTS = [359]
+
+
+def title(title, underline='='):
+    print(title)
+    print(underline * len(title))
+
+
+def list(items, provider, indent=''):
```

Comment:
list is a built-in name. It might confuse the readers. It's not forbidden to use it though, but it's better to rename the function.